types/mui-datatables/index.d.ts

```diff
@@ -12,38 +12,37 @@ import * as React from 'react';
 export type Display = 'true' | 'false' | 'excluded';
 export type SortDirection = 'asc' | 'desc';
 export type FilterType = 'dropdown' | 'checkbox' | 'multiselect' | 'textField' | 'custom';
-export type Responsive = 'stacked' | 'scroll';
+export type Responsive = 'stacked' | 'scrollMaxHeight' | 'scrollFullHeight';
```

Comment:
Why is this re-ordered?

Reply:
New options added for the Responsive type 'scroll' was removed. Added these in the order they appeared on the https://github.com/gregnb/mui-datatables